cmd/crc/cmd/config/config.go

```diff
@@ -19,6 +19,8 @@ var (
 	NameServer     = cfg.AddSetting("nameserver", nil, []cfg.ValidationFnType{cfg.ValidateIpAddress}, []cfg.SetFn{cfg.SuccessfullyApplied})
 	PullSecretFile = cfg.AddSetting("pull-secret-file", nil, []cfg.ValidationFnType{cfg.ValidatePath}, []cfg.SetFn{cfg.SuccessfullyApplied})
 
+	NoUpdateCheck = cfg.AddSetting("no-update-check", nil, []cfg.ValidationFnType{cfg.ValidateBool}, []cfg.SetFn{cfg.SuccessfullyApplied})
```

Comment:
`disable` probably sounds better
since this allows to also use `enable` as lingo.

there is no good opposite in this context for `no`; eg. `yes-telemetry`

Reply:
The opposite of `no` is to just drop it, `--update-check` `--no-update-check`. But I almost suggested using disable/enable before, so not objecting to your suggestion ;)